lib/components/src/toolbar/toolbar.ts

```diff
@@ -0,0 +1,57 @@
+import { customElement, property } from "lit/decorators.js";
+import styles from './toolbar.scss';
+import { TopAppBarBase } from '@material/mwc-top-app-bar/mwc-top-app-bar-base';
+import { html } from "lit";
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'td-toolbar': CovalentToolbarBase,
+  }
+}
+
+@customElement('td-toolbar')
+export class CovalentToolbarBase extends TopAppBarBase {
+  @property({type: Number}) breadcrumbs = 0;
+  
+  static override styles = [styles];
+  override render() {
+    // Create dummy data for the toolbar breadcrumbs.
+    let path: any[] = [];
+    if(this.breadcrumbs > 5) {
+      path.push(html`<span>Section title <td-icon>chevron_right</td-icon> ... <td-icon>chevron_right</td-icon> </span>`);
+      path.push('Page title');
+    }
+    else {
+      for(let i = 0; i < this.breadcrumbs; i++) {
+        if(i == this.breadcrumbs-1) {
+          path.push(`Page title`);
+        }
+        else {
+          path.push(html`<span>Section title <td-icon>chevron_right</td-icon> </span>`);
+        }
+      }
+    }
```

Comment:
Is there a way to specify the page title or section titles?

Reply:
I'm not sure. Maybe if each section is a new page then the URL can be parsed for the path.